docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -16,54 +16,55 @@ An Expectation is a verifiable assertion about your data. They make implicit ass
-| Validity           | **column values to match regex list**                | Expect the column entries to be strings that can be matched to either any of or all of a list of regular expressions.                  | No                  | No                |
-| Validity           | **column values to not match like pattern**          | Expect the column entries to be strings that do NOT match a given like pattern expression.                                             | No                  | No                |
-| Validity           | **column values to not match like pattern list**     | Expect the column entries to be strings that do NOT match any of a provided list of like pattern expressions.                          | No                  | No                |
-| Validity           | **column values to not match regex**                 | Expect the column entries to be strings that do NOT match a given regular expression.                                                  | No                  | No                |
-| Validity           | **column values to not match regex list**            | Expect the column entries to be strings that do not match any of a list of regular expressions. Matches can be anywhere in the string. | No                  | No                |
-| Volume             | **table row count to be between**                    | Expect the number of rows to be between two values.                                                                                    | Yes                 | Yes                |
-| Volume             | **table row count to equal**                         | Expect the number of rows to equal a value.                                                                                            | No                  | No                |
-| Volume             | **table row count to equal other table**             | Expect the number of rows to equal the number in another table or view within the same database.                                       | No                  | No                |
+| Data quality issue | Expectation                                            | Description                                                                                                                            | Dynamic Parameters? | Forecasted range? |
+|--------------------|--------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------|---------------------|-------------------|
+| Completeness       | **column proportion of non-null values to be between** | Expect the proportion of non-null values to be between a minimum value and a maximum value.                                            | Yes                 | No                |
```

Comment:
Note for reviewer: The diff of this table looks terrible but the only substantial change is this new row. the rest of the changes should just be whitespace changes